### PR TITLE
uni07eta dt  - deploys networking nodes and Cinder (iSCSI)

### PR DESCRIPTION
**Pull request by arxcruz**
Uni07eta - Providing deployment topology

### CODE REVIEW

File: dt/uni07eta/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Deployed Topology - Eta
+
+If you are looking for information on how to deploy the alpha based DT, then
```
Comment (sdatko) [inline dt/uni07eta/README.md:3]: s/alpha/eta/ probably?

File: dt/uni07eta/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+please the [README](../../examples/dt/uni07eta/README.md) in the examples
```
Comment (sdatko) [inline dt/uni07eta/README.md:4]: please **SEE** the ... ?

File: dt/uni07eta/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Deployed Topology - Eta
```
Comment (sdatko) [inline dt/uni07eta/README.md:1]: According to the main readme, it should be `deployment` probably?

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+      availability-zones:
+        - zone-1
+
+  nova:
+    schedulerServiceTemplate:
+      customServiceConfig: |
+        [filter_scheduler]
+        enabled_filters = AggregateInstanceExtraSpecsFilter, AvailabilityZoneFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,DifferentHostFilter,SameHostFilter,AllHostsFilter,ServerGroupAffinityFilter,ServerGroupAntiAffinityFilter
+
+  neutron:
```
Comment (fyanac) [inline examples/dt/uni07eta/control-plane/service-values.yaml:84]: For Uni07eta job, it should be the neutron config of vlan_provider.yaml: 
https://github.com/openstack-k8s-operators/architecture/pull/73/files#diff-2c13260a55330f998d004019e04c0a74928394530e1cf409bf775aff35049412

File: examples/va/hci/edpm-pre-ceph/values.yaml [hunk 5]
```diff
@@ -47,6 +47,10 @@ data:
           {%- endfor %}
           {% set min_viable_mtu = mtu_list | max %}
           network_config:
+          - type: interface
+            name: nic1
+            use_dhcp: true
+            mtu: {{ min_viable_mtu }}
```
Comment (abays) [inline examples/va/hci/edpm-pre-ceph/values.yaml:53]: Is this change supposed to be included here?  It's changing a VA, but this PR is adding a new DT.

File: examples/va/nfv/ovs-dpdk-sriov/service-values.yaml [hunk 6]
```diff
@@ -24,6 +24,14 @@ data:
...
       [ml2_type_vlan]
       network_vlan_ranges = dpdk1:206:209,dpdk2:206:209,sriov1,sriov2 # CHANGEME
+      [neutron]
+      physnets = dpdk1, dpdk2 # CHANGEME
+      [neutron_physnet_dpdk1] # CHANGEME
+      numa_nodes = 0 # CHANGEME
+      [neutron_physnet_dpdk2 # CHANGEME
+      numa_nodes = 0 # CHANGEME
+      [neutron_tunnel] # CHANGEME
+      numa_nodes = 0 # CHANGEME
```
Comment (abays) [inline examples/va/nfv/ovs-dpdk-sriov/service-values.yaml:34]: Same question here.  Is this change supposed to be included here?  It's changing a VA, but this PR is adding a new DT.

Comment (dsariel) [reply examples/va/nfv/ovs-dpdk-sriov/service-values.yaml:34]: I thought it will not harm to have this. Thanks for pointing. Removing the change

File: examples/va/hci/edpm-pre-ceph/values.yaml (hunk 5 above)
Comment (dsariel) [reply examples/va/hci/edpm-pre-ceph/values.yaml:53]: I thought it will not harm to have this. Thanks for pointing. Removing the change

File: lib/control-plane/openstackcontrolplane.yaml [hunk 7]
```diff
@@ -182,3 +182,27 @@ spec:
...
+            strategy: persistent
+            retention: 24h
+            persistent:
+              pvcStorageRequest: 20G
+      autoscaling:
+        enabled: false
+        aodh:
+          passwordSelectors:
+          databaseInstance: openstack
+          memcachedInstance: memcached
```
Comment (abays) [inline lib/control-plane/openstackcontrolplane.yaml:203]: These two lines need to be indented

File: examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml [hunk 8]
```diff
@@ -194,14 +194,6 @@ data:
...
         cpu_shared_set = 0-3,24-27
         cpu_dedicated_set = 8-23,32-47
-        [neutron]
-        physnets = dpdk1, dpdk2 # CHANGEME
-        [neutron_physnet_dpdk1] # CHANGEME
-        numa_nodes = 0 # CHANGEME
-        [neutron_physnet_dpdk2] # CHANGEME
-        numa_nodes = 0 # CHANGEME
-        [neutron_tunnel] # CHANGEME
-        numa_nodes = 0 # CHANGEME
```
Comment (abays) [inline examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml:204]: Why are these lines being removed?  They seem unrelated to the DT.

File: examples/va/nfv/ovs-dpdk/edpm/values.yaml [hunk 9]
```diff
@@ -174,14 +174,6 @@ data:
...
         cpu_shared_set = 0-3,24-27
         cpu_dedicated_set = 8-23,32-47
-        [neutron]
-        physnets = dpdk1, dpdk2 # CHANGEME
-        [neutron_physnet_dpdk1] # CHANGEME
-        numa_nodes = 0 # CHANGEME
-        [neutron_physnet_dpdk2] # CHANGEME
-        numa_nodes = 0 # CHANGEME
-        [neutron_tunnel] # CHANGEME
-        numa_nodes = 0 # CHANGEME
```
Comment (abays) [inline examples/va/nfv/ovs-dpdk/edpm/values.yaml:184]: Same here: why are these lines being removed?  They seem unrelated to the DT.

File: test-kustomizations.sh [hunk 10]
```diff
@@ -60,7 +60,14 @@ mapfile -t FILES < <(find "${DIRS_TO_CHECK[@]}" \
 for FILE in "${FILES[@]}"; do
     DIRECTORY=$(dirname "${FILE}")
 
-    if ! OUTPUT=$("${COMMAND}" build "${DIRECTORY}" 2>&1); then
+    if [ $(grep -ce "^${DIRECTORY}$" .ci/relax_loader.txt) -eq 1 ]; then
```
Comment (abays) [inline test-kustomizations.sh:63]: When I run this locally, it need to change `[ ]` to `[[ ]]` or else I get:

```
../test-kustomizations.sh: line 63: [: -eq: unary operator expected
```

Comment (abays) [reply test-kustomizations.sh:63]: Also, even after the fix, the output is somewhat verbose:

```
grep: .ci/relax_loader.txt: No such file or directory
./common/metallb/kustomization.yaml: OK
grep: .ci/relax_loader.txt: No such file or directory
./common/nmstate/kustomization.yaml: OK
grep: .ci/relax_loader.txt: No such file or directory
./common/olm/kustomization.yaml: OK
```

Maybe we could suppress the `grep` message, or do you think that is desirable to see/know?

Comment (tosky) [reply test-kustomizations.sh:63]: Which shell do you use?

Comment (sdatko) [reply test-kustomizations.sh:63]: Please do not look at this file. I guess David copied it from Pragadeeswaran, but I already requested changes there and it has better shape now: https://github.com/openstack-k8s-operators/architecture/pull/83/files#diff-1660d5268f2b4dcabf03d84bd0caf8486d4e73ded11d38920618e1fdd2bc78b4

Comment (abays) [reply test-kustomizations.sh:63]: Understood.  Could we remove this from the PR, then?

Comment (sdatko) [reply test-kustomizations.sh:63]: TL;DR This change does not belong here; @dsariel please restore the file to the shape from current `main` branch

Comment (dsariel) [reply test-kustomizations.sh:63]: changed

File: examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml (hunk 8 above)
Comment (dsariel) [reply examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml:204]: reverted

File: lib/control-plane/openstackcontrolplane.yaml (hunk 7 above)
Comment (dsariel) [reply lib/control-plane/openstackcontrolplane.yaml:203]: done

File: test-kustomizations.sh (hunk 10 above)
Comment (sdatko) [reply test-kustomizations.sh]: I still see it in current commit @dsariel :wink:

File: test-kustomizations.sh [hunk 11]
```diff
@@ -60,7 +60,14 @@ mapfile -t FILES < <(find "${DIRS_TO_CHECK[@]}" \
...
 
-    if ! OUTPUT=$("${COMMAND}" build "${DIRECTORY}" 2>&1); then
+    if [[ $(grep -ce "^${DIRECTORY}$" .ci/relax_loader.txt) -eq 1 ]]; then
+        arg="--load-restrictor=LoadRestrictionsNone"
+        OUTPUT=$("${COMMAND}" build "${arg}" "${DIRECTORY}" 2>&1)
+    else
+        OUTPUT=$("${COMMAND}" build "${DIRECTORY}" 2>&1)
+    fi
+
+    if [[ ! ${OUTPUT} ]] ; then
```
Comment (sdatko) [inline test-kustomizations.sh:70]: `git checkout HEAD~1 -- test-kustomizations.sh`

Comment (dsariel) [reply test-kustomizations.sh:70]: Based on our chat I guess you mean: git rm test-kustomizations.sh

Comment (dsariel) [reply test-kustomizations.sh]: @sdatko please disregard my last comment. I read the thread bw you and @abays and restoring the previous version of test-kustomizations.sh

File: examples/dt/uni07eta/control-plane.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+oc project openstack
+```
+
+Change to the uni07eta directory
+
+```bash
+cd architecture/examples/dt/uni07eta
+```
+
+Edit [service-values.yaml](service-values.yaml) and
```
Comment (fultonj) [inline examples/dt/uni07eta/control-plane.md:21]: the path to service-values.yaml is wrong (it needs the control-plane directory)
```
[fultonj@runcible uni07eta{uni-etha}]$ pwd
/home/fultonj/review/architecture/examples/dt/uni07eta
[fultonj@runcible uni07eta{uni-etha}]$  find . -name service-values.yaml
./control-plane/service-values.yaml
[fultonj@runcible uni07eta{uni-etha}]$ 
```

File: examples/dt/uni07eta/control-plane/nncp/values.yaml [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+          - end: 192.168.122.120
+            start: 192.168.122.100
+          - end: 192.168.122.200
+            start: 192.168.122.150
+        cidr: 192.168.122.0/24
+        gateway: 192.168.122.1
+        name: subnet1
+    prefix-length: 24
+    iface: enp6s0
+    mtu: 9000
```
Comment (fultonj) [inline examples/dt/uni07eta/control-plane/nncp/values.yaml:44]: Why MTU 9000 on the control plane?

See https://github.com/openstack-k8s-operators/architecture/pull/159#discussion_r1571107733

File: examples/dt/uni07eta/control-plane/nncp/values.yaml [hunk 14]
```diff
@@ -0,0 +1,212 @@
...
+        "type": "macvlan",
+        "master": "storage",
+        "ipam": {
+          "type": "whereabouts",
+          "range": "172.18.0.0/24",
+          "range_start": "172.18.0.30",
+          "range_end": "172.18.0.70"
+        }
+      }
+  storagemgmt:
```
Comment (fultonj) [inline examples/dt/uni07eta/control-plane/nncp/values.yaml:126]: Why do you need the storagemgmt network here? If you're hosting Ceph on EDPM nodes you need it. What is it being used for here?

File: examples/dt/uni07eta/networker.md [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+
+# 3. Modify [networker/values.yaml](networker/values.yaml) file to suit your
+#    environment.
+
+    $ pushd networker
+    $ vi values.yaml
+
+# 4. Generate the networker data plane deployment plan.
+
+    $ kustomize build --load-restrictor LoadRestrictionsNone > \
```
Comment (fultonj) [inline examples/dt/uni07eta/networker.md:27]: Can we build the `edpm-networker.yaml` without `--load-restrictor LoadRestrictionsNone`? 

Right now it has no effect on the default samples.

```
[fultonj@runcible uni07eta{uni-etha}]$ pwd
/home/fultonj/review/architecture/examples/dt/uni07eta
[fultonj@runcible uni07eta{uni-etha}]$ kustomize build > one.yaml
[fultonj@runcible uni07eta{uni-etha}]$  kustomize build --load-restrictor LoadRestrictionsNone > two.yaml
[fultonj@runcible uni07eta{uni-etha}]$ md5sum one.yaml two.yaml 
66ae6992f58646c683d36b7a3dc51b42  one.yaml
66ae6992f58646c683d36b7a3dc51b42  two.yaml
[fultonj@runcible uni07eta{uni-etha}]$
```

File: examples/dt/uni07eta/README.md [hunk 16]
```diff
@@ -0,0 +1,258 @@
...
+#### iSCSI
+
+It is assumed *iSCSI* services are enabled in all nodes participating in the
+Red Hat OpenShift cluster. If not, a `MachineConfig` similar to the below one
+is applied. The node would be *rebooted* after applying the configuration.
+
+```YAML
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
```
Comment (fultonj) [inline examples/dt/uni07eta/README.md:85]: I have not tried `MachineConfig`. I'm relying on people in the cinder group to review this part.

File: lib/control-plane/openstackcontrolplane.yaml [hunk 17]
```diff
@@ -182,3 +182,27 @@ spec:
         ringReplicas: 1
       swiftStorage:
         replicas: 1
+  telemetry:
```
Comment (fultonj) [inline lib/control-plane/openstackcontrolplane.yaml:185]: Do we want this in the lib/control-plane that ALL VAs/DTs use?

Perhaps it should be a file that this DT can access (and others) but not mandatory for all. I think this is similar Andrew's recent comment about `lib/control-plane/kustomization.yaml`

https://github.com/openstack-k8s-operators/architecture/pull/83#discussion_r1570943223

**Review by fultonj — COMMENTED** [review]
Looks OK, overall. Thanks. There's a few things which could be improved before merging though.

File: lib/control-plane/openstackcontrolplane.yaml (hunk 17 above)
Comment (sdatko) [reply lib/control-plane/openstackcontrolplane.yaml:185]: Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

File: examples/dt/uni07eta/README.md (hunk 16 above)
Comment (sdatko) [reply examples/dt/uni07eta/README.md:85]: Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

File: examples/dt/uni07eta/control-plane/nncp/values.yaml (hunk 14 above)
Comment (sdatko) [reply examples/dt/uni07eta/control-plane/nncp/values.yaml:126]: Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

File: examples/dt/uni07eta/control-plane/nncp/values.yaml (hunk 13 above)
Comment (sdatko) [reply examples/dt/uni07eta/control-plane/nncp/values.yaml:44]: Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

File: examples/dt/uni07eta/control-plane.md (hunk 12 above)
Comment (sdatko) [reply examples/dt/uni07eta/control-plane.md:21]: Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

File: examples/dt/uni07eta/README.md (hunk 16 above)
Comment (arxcruz) [reply examples/dt/uni07eta/README.md:85]: This is working, according @psathyan

File: examples/dt/uni07eta/control-plane/nncp/values.yaml (hunk 14 above)
Comment (fultonj) [reply examples/dt/uni07eta/control-plane/nncp/values.yaml:126]: 83 is merged please rebase

File: dt/uni07eta/kustomization.yaml [hunk 18]
```diff
@@ -0,0 +1,439 @@
...
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.redis.enabled
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.redis.enabled
+        options:
+          create: true
```
Comment (psathyan) [inline dt/uni07eta/kustomization.yaml:426]: We can remove this as it is not used.

File: dt/uni07eta/networker/kustomization.yaml [hunk 19]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (psathyan) [inline dt/uni07eta/networker/kustomization.yaml:1]: IMHO, we should split this into `nodeset` and `deployment`.

File: examples/dt/uni07eta/control-plane.md [hunk 20]
```diff
@@ -0,0 +1,88 @@
...
+  namespace: openshift-operators
+  labels:
+    operators.coreos.com/observability-operator.openshift-operators: ""
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: cluster-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
```
Comment (psathyan) [inline examples/dt/uni07eta/control-plane.md:30]: This is required for `telemetry` operator. Based on the `kustomization`, I noticed we are not enabling `telemetry` hence we could remove this lines.

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 21]
```diff
@@ -0,0 +1,173 @@
...
+      customServiceConfig: |
+        [controller_worker]
+        loadbalancer_topology=ACTIVE_STANDBY
+
+
+  heat:
+    enabled: true
+
+  redis:
+    enabled: true
```
Comment (psathyan) [inline examples/dt/uni07eta/control-plane/service-values.yaml:122]: these lines can be removed.

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 22]
```diff
@@ -0,0 +1,173 @@
...
+
+
+  ovn:
+    ovnController:
+      nicMappings:
+        datacentre: ocpbr
+        octavia: octbr
+      availability-zones:
+        - zone-1
```
Comment (psathyan) [inline examples/dt/uni07eta/control-plane/service-values.yaml:131]: this isn't required here as we are not having OVN Availability Zones.

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 23]
```diff
@@ -0,0 +1,173 @@
...
+  neutron:
+    customServiceConfig: |
+      [DEFAULT]
+      vlan_transparent = true
+      agent_down_time = 600
+      router_distributed = true
+      router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
+      allow_automatic_l3agent_failover = true
+      debug = true
+      default_availability_zones = zone-1,zone-2
```
Comment (psathyan) [inline examples/dt/uni07eta/control-plane/service-values.yaml:142]: Likewise, we don't need this line as we are not focused on OVN Availability zones.

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 4 above)
Comment (fyanac) [reply examples/dt/uni07eta/control-plane/service-values.yaml:133]: Hi, 
The Uni07eta job, needs to be updated with neutron config with the link above(vlan_provider) again.
It seems it has alpha config.

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 23 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane/service-values.yaml:142]: done

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 22 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane/service-values.yaml:131]: done

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 21 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane/service-values.yaml:122]: done

File: examples/dt/uni07eta/control-plane.md (hunk 20 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane.md:30]: done

File: dt/uni07eta/kustomization.yaml (hunk 18 above)
Comment (dsariel) [reply dt/uni07eta/kustomization.yaml:426]: done

File: automation/vars/uni07eta.yaml [hunk 24]
```diff
@@ -0,0 +1,53 @@
...
+vas:
+  uni07eta:
+    stages:
+      - path: examples/dt/uni07eta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
```
[the remark this answers is not in the document]
Comment (dsariel) [reply automation/vars/uni07eta.yaml:11]: done. However, other unijobs define 60s. Should we change in all jobs?

File: examples/dt/uni07eta/control-plane/nncp/values.yaml (hunk 14 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane/nncp/values.yaml]: marking as resolved

File: examples/dt/uni07eta/control-plane.md (hunk 12 above)
Comment (dsariel) [reply examples/dt/uni07eta/control-plane.md]: resolved. Thanks!

File: lib/control-plane/openstackcontrolplane.yaml (hunk 17 above)
Comment (dsariel) [reply lib/control-plane/openstackcontrolplane.yaml]: marking as resolved for uni07 change.

File: automation/vars/uni07eta.yaml (hunk 24 above)
[the remark this answers is not in the document]
Comment (abays) [reply automation/vars/uni07eta.yaml]: I believe it should indeed by doubled or tripled across all jobs

[the remark this answers is not in the document]
Comment (dsariel) [reply automation/vars/uni07eta.yaml]: Thanks a lot @abays . Created https://github.com/openstack-k8s-operators/architecture/pull/265

File: dt/uni07eta/edpm/kustomization.yaml [hunk 25]
```diff
@@ -0,0 +1,20 @@
+---
```
[the remark this answers is not in the document]
Comment (dsariel) [reply dt/uni07eta/edpm/kustomization.yaml:1]: spitted

File: dt/uni07eta/networker/kustomization.yaml (hunk 19 above)
Comment (dsariel) [reply dt/uni07eta/networker/kustomization.yaml:1]: splitted

**Comment by sdatko** [comment]
Ready for review :+1:

File: automation/vars/uni07eta.yaml [hunk 26]
```diff
@@ -0,0 +1,76 @@
...
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackdataplanedeployment
+            networker-deploy
+            --for condition=Ready
+            --timeout=40m
+        values:
+          - name: edpm-deployment-values
+            src_file: values.yaml
+        build_output: edpm-networker.yaml
```
Comment (abays) [inline automation/vars/uni07eta.yaml:53]: It might be prudent to differentiate the output file name on L41 and L53

File: automation/vars/uni07eta.yaml [hunk 27]
```diff
@@ -0,0 +1,76 @@
...
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackdataplanedeployment
+            edpm-deployment
+            --for condition=Ready
+            --timeout=40m
+        values:
+          - name: edpm-deployment-values
+            src_file: values.yaml
+        build_output: edpm.yaml
```
Comment (abays) [inline automation/vars/uni07eta.yaml:76]: Same thing here on L65 and L76 for output file name

File: examples/dt/uni07eta/control-plane/nncp/values.yaml (hunk 13 above)
Comment (abays) [reply examples/dt/uni07eta/control-plane/nncp/values.yaml:45]: Is Ceph being used?  Do we need an MTU of 9000?

File: automation/vars/uni07eta.yaml (hunk 26 above)
Comment (sdatko) [reply automation/vars/uni07eta.yaml]: Fixed

File: automation/vars/uni07eta.yaml (hunk 27 above)
Comment (sdatko) [reply automation/vars/uni07eta.yaml]: Fixed

**Review by karelyatin — COMMENTED** [review]
Also noticed some comments are marked as resolved with comments Done but not updated

File: examples/dt/uni07eta/control-plane/nncp/values.yaml (hunk 13 above)
Comment (fyanac) [reply examples/dt/uni07eta/control-plane/nncp/values.yaml:45]: Hi, Talking with neutron-qe we no longer need mtu 9000 any more, but I'm not sure if storage needs.

File: dt/uni07eta/kustomization.yaml [hunk 28]
```diff
@@ -0,0 +1,319 @@
...
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.octavia.template.octaviaWorker.networkAttachments
+        options:
+          create: true
+  # Redis
+  - source:
+      kind: ConfigMap
+      name: service-values
```
[the remark this answers is not in the document]
Comment (sdatko) [reply dt/uni07eta/kustomization.yaml:262]: As far as we know it is required for Octavia. Do you say it is not?

File: examples/dt/uni07eta/control-plane/service-values.yaml [hunk 29]
```diff
@@ -0,0 +1,131 @@
...
+
+    default:
+      replicas: 1
+
+  swift:
+    enabled: true
+
+
+  octavia:
+    enabled: false
```
[the remark this answers is not in the document]
Comment (sdatko) [reply examples/dt/uni07eta/control-plane/service-values.yaml:62]: It is kept disabled in all other DTs at the moment.

It will be enabled after the following change is merged: https://github.com/openstack-k8s-operators/ci-framework/pull/1756/files

File: dt/uni07eta/kustomization.yaml (hunk 28 above)
[the remark this answers is not in the document]
Comment (karelyatin) [reply dt/uni07eta/kustomization.yaml:262]: Redis is no longer a thing since https://github.com/openstack-k8s-operators/openstack-operator/pull/748, so all references can be dropped

**Comment by fultonj** [comment]
please ensure the commit message and PR message are updated to not say "do not merge"

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 29 above)
[the remark this answers is not in the document]
Comment (karelyatin) [reply examples/dt/uni07eta/control-plane/service-values.yaml:62]: ok i now see we have https://github.com/openstack-k8s-operators/architecture/commit/2a955bf6e13b01d94a11ab0998357429e85e5f34, would have been better to point to relevant issues whenever we disable things so context is at the same place of where it's disabled and avoid clarifications.

File: examples/dt/uni07eta/control-plane/service-values.yaml (hunk 4 above)
Comment (sdatko) [reply examples/dt/uni07eta/control-plane/service-values.yaml:91]: @dsariel Would you update the Neutron config here or as a separate change?

File: dt/uni07eta/kustomization.yaml (hunk 28 above)
[the remark this answers is not in the document]
Comment (sdatko) [reply dt/uni07eta/kustomization.yaml:262]: Ok, then we have to remove it also from Alpha and Epsilon, I will take care of it there :+1:

[the remark this answers is not in the document]
Comment (dsariel) [reply dt/uni07eta/kustomization.yaml:262]: done

**Comment by dsariel** [comment]
@karelyatin added another commit that will be squashed into the original once the test is passing. This one related to removal of redis and availability zones as marked done previously.

**Comment by fultonj** [comment]
@dsariel 

1. Can you update so we don't have this issue
```

**Review by karelyatin — COMMENTED** [review]
/lgtm

Thanks
